### PR TITLE
zm: add page

### PR DESCRIPTION
**Pull request by pixelcmtd**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/zm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Make a new draft:
+
+`zm new`
+
+- Edit a draft:
+
+`zm edit`
+
+- Publish a draft:
```
Comment (blueskyson) [inline pages/common/zm.md:14]: I've read `zm` shell script, `zm publish` means commit a draft to git right? Mention git commit here is clearer.

File: pages/common/zm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`zm publish`
```
Comment (blueskyson) [inline pages/common/zm.md:16]: I've read `zm` shell script, `zm publish` means commit a draft to git right? Mention git commit here is clearer.

**Comment by marchersimon** [comment]
Does this something have to do with `lb`? It might be good to add a ``See also: `lb` `` note here and one for `zb` in `lb.md`.

**Comment by chrissxYT** [comment]
> Does this something have to do with `lb`? It might be good to add a `` See also: `lb` `` note here and one for `zb` in `lb.md`.

Not anymore. `zm1` was a fork of `lb`, but `zm2` is written from scratch and only "inspired" by the way `zm1` and `lb` work.

**Comment by marchersimon** [comment]
We often reference commands which do similar things, even though they don't have a common codebase.

**Review by blueskyson — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @pixelcmtd

**Pull request merged**